### PR TITLE
NO-DED corrections reported on platform docs

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description
Two different issues reported on `platform-docs` are fixed here

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/authorization/authorizing-api-calls/index.mdx [hunk 1]
```diff
@@ -763,7 +763,7 @@ Your web browser will automatically store and attach these cookies to future req
 
 ```shell
 curl "https://accounts.livechat.com/customer/token" \
-  -X POST
+  -X POST \
```
Comment (kamilatokarska) [inline src/pages/authorization/authorizing-api-calls/index.mdx:766]: Hello @bolchowka :) I would appreciate it if you could just confirm that these are the correct changes connected with the case reported on [platform-docs](https://livechat.slack.com/archives/C2XMTMQ5S/p1658306967536979). I am fixing two issues in this PR, so please do not pay attention to the above change. Thanks :)

**Comment by bolchowka** [comment]
Hey @kamilatokarska, looks good!

**Review by bolchowka — APPROVED** [review]

**Review by oliwiapolec — APPROVED** [review]

File: src/configs/redoc/global-accounts-api/spec.yml [hunk 2]
```diff
@@ -1527,8 +1528,7 @@ paths:
         revoked.
       security:
         - OAuth2 Bearer Token:
-            - accounts--all:ro
-            - accounts.roles--all:ro
+            - sessions-my:rw
```
Comment (kamilatokarska) [inline src/configs/redoc/global-accounts-api/spec.yml:1531]: @przemekdudycz - case from [platform-docs](https://livechat.slack.com/archives/C2XMTMQ5S/p1658241447403929)

File: src/configs/redoc/global-accounts-api/spec.yml [hunk 3]
```diff
@@ -414,12 +414,13 @@ paths:
       description: >-
         Get an account for a given `account_id`. The ID can be replaced with
```
Comment (kamilatokarska) [inline src/configs/redoc/global-accounts-api/spec.yml:415]: @przemekdudycz - case from [platform-docs](https://livechat.slack.com/archives/C2XMTMQ5S/p1658484657302999)

**Pull request merged**